mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,20 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.removeOrder = (req, res) => { 
+  Order.findByIdAndDelete(req.params.orderId) 
+    .then(deletedOrder => { 
+      if (!deletedOrder) {
```

Comment:
Inconsistent spacing in the if statement. There should be a space between the `if` keyword and the opening parenthesis. It should be `if (!deletedOrder) {` instead of `if (!deletedOrder){`.